3.3-apache/Dockerfile

```diff
@@ -28,11 +45,11 @@ RUN apt-get update \
  && sed -i /etc/modsecurity.d/modsecurity.conf \
       -e 's/SecRuleEngine DetectionOnly/SecRuleEngine On/g'
 
-COPY httpd-logging-before-modsec.conf /usr/local/apache2/conf/extra/
-COPY httpd-logging-after-modsec.conf  /usr/local/apache2/conf/extra/
```

Comment:
I'm not sure whether we'll still need those files.

In [vshn/modsecurity](https://github.com/vshn/modsecurity-docker/blob/master/v3.1/conf-available/logging.conf) we have all the logging configuration in a single configuration file.